src/test/java/com/browserstack/utils/CapabilityConverter.java

```diff
@@ -0,0 +1,23 @@
+package com.browserstack.utils;
+
+import org.junit.jupiter.api.extension.ParameterContext;
+import org.junit.jupiter.params.aggregator.ArgumentsAccessor;
+import org.junit.jupiter.params.aggregator.ArgumentsAggregationException;
+import org.junit.jupiter.params.aggregator.ArgumentsAggregator;
+import org.openqa.selenium.remote.DesiredCapabilities;
+
+public class CapabilityConverter implements ArgumentsAggregator {
```

Comment:
the class in its current state requires a 1 line description that its objective is to create the desiredCapabilities object from raw data of caps which comes in the format <cap_name>:<cap_value>

any other option to make the objective clear without having to add a line of comment? As the last alternative, a comment line is fine.